catroid/src/catroid/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -37,11 +37,17 @@
 
 	public static final String POCKET_CODE_EXTERNAL_STORAGE_FOLDER_NAME = "Pocket Code";
 
+	public static final String POCKET_CODE_EXTERNAL_EXPORT_STORAGE_FOLDER_NAME = "CATROBAT";
+
+	public static final String POCKET_CODE_ZIP_EXTENSION = ".CATROBAT";
```

Comment:
are you sure you want to have these constants flavoured? i.e. are these going to be different for each flavour? e.g. is luna and cat going to have a different extension, which will prevent you from opening projects from other flavours?

and besides: does this constant not already exist in `Constants.java`?